magprime/templates/regextra.html

```diff
@@ -0,0 +1,29 @@
+<script type="text/javascript">
+    $(function () {
```

Comment:
Yeah, I'd lean more towards having fewer magic custom utility functions - it makes inspecting code/reading across errors more difficult.